WORKSPACE

```diff
@@ -141,3 +134,54 @@ http_archive(
     strip_prefix = "googleapis-143084a2624b6591ee1f9d23e7f5241856642f4d",
     urls = ["https://github.com/googleapis/googleapis/archive/143084a2624b6591ee1f9d23e7f5241856642f4d.zip"],
 )
```

Comment:
`@googleapis` is extra over bb-storage's `WORKSPACE`; AFAICT this is because `@googleapis` is required for `remote-asset-apis` but bb-storage patches remote-asset-apis to use `@go_googleapis` instead sufficiently for its usecase.

Either the patch to remote-asset-apis needs to be reworked to cover more of the protos, or maybe remote-asset-apis can be fixed in a way that doesn't necessitate the patch, but having this here doesn't seem to be causing build issues yet.